### ISSUE TITLE
Run on original Modded XBOX?

### ISSUE THREAD

**Issue by cfredericksen** [lostsoul63b/NFL-2K23#5]
I play NFL2k5 on my original XBOX with updated rosters. Can I run the NFL2k24 download from a modded xbox if FTP the files over?
@lostsoul63b

**Comment by BAD-AL** [comment]
Yes, there should be able to FTP those Saves over to your Modded OG XBOX as normal.
And you should also be able to do that in XEMU if you get networking setup done according to their networking guidelines (using something like the Hexen 2018 build shown in many YouTube tutorials).
